tools/wpt/utils.ts

```diff
@@ -14,12 +14,26 @@ export const {
   ["auto-config"]: autoConfig,
   ["inspect-brk"]: inspectBrk,
   binary,
+  shard: shardStr,
 } = parse(Deno.args, {
   "--": true,
   boolean: ["quiet", "release", "no-interactive", "inspect-brk"],
-  string: ["json", "wptreport", "binary"],
+  string: ["json", "wptreport", "binary", "shard"],
```

Comment:
I can't find any reference to this arg, either I made a mistake during rebases, this arg is supported internally by WPT suites or it's not working as expected